crates/ra_ide/src/completion/presentation.rs

```diff
@@ -273,12 +273,12 @@ impl Completions {
             .map(|field| (field.name(ctx.db), field.signature_ty(ctx.db)));
         let variant_kind = variant.kind(ctx.db);
         let detail = match variant_kind {
-            StructKind::Tuple | StructKind::Unit => detail_types
+            StructKind::Tuple(_) | StructKind::Unit => detail_types
```

Comment:
I wonder if it makes sense to bind `_` here, instead of using `detail_types` from above? We are ignoring field names anyway in this branch, so this copy-paste eliminaiton feels like false sharing.

Reply:
I'm not sure what you mean here. Are you suggesting deleting `detail_types` above and doing the `variant.fields()..` in the match arms?

Reply:
```
match variant_kind {
  StructKind::Tuple(field) => fields
}
```

Reply:
fwiw, I don't think it's necessary necessary to change  this :) 

bors d+

(I think this needs a rebase after a copule of recent renames?)